handlers/services.go

```diff
@@ -130,6 +131,40 @@ func ServiceDetails(w http.ResponseWriter, r *http.Request) {
 	RespondWithJSON(w, http.StatusOK, serviceDetails)
 }
 
+func ServiceUpdate(w http.ResponseWriter, r *http.Request) {
+	// Get business layer
+	business, err := getBusiness(r)
+	if err != nil {
+		RespondWithError(w, http.StatusInternalServerError, "Services initialization error: "+err.Error())
+		return
+	}
+
+	params := mux.Vars(r)
+	namespace := params["namespace"]
+	service := params["service"]
+
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		RespondWithError(w, http.StatusBadRequest, "Update request with bad update patch: "+err.Error())
```

Comment:
I guess a `return` is missing here?

Reply:
right, and probably from the place I copied :-) too.